### PR TITLE
Add guidelines for use of login shells

### PR DESCRIPTION
**Pull request by thomaspaulmann**
An update to our guidelines for the use of login shells. I added the information to the contribution guidelines as well as added an hint to the README to make sure that new contributors now how to handle it.

Additionally I created #77 to keep track of our work on support for env variables.

—

Created via [Raycast](https://www.raycast.com?ref=signatureGithub)

### CODE REVIEW

**Review by PitNikola — APPROVED** [review]
👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ While `packageName` is an optional parameter and if it's missing Raycast will de
 
 Example: `github-notifications.template.sh`
 
-NOTE: This might change as soon as we introduce a better way to provide parameters / enviromental variables.
+*NOTE:* This might change as soon as we introduce a better way to provide parameters / enviromental variables.
```
Comment (fe9lix) [inline CONTRIBUTING.md:49]: enviro**n**mental

Comment (thomaspaulmann) [reply CONTRIBUTING.md:49]: Good catch!

Comment (thomaspaulmann) [reply CONTRIBUTING.md:49]: Fixed!

Comment (unnamedd) [reply CONTRIBUTING.md:49]: @fe9lix hawkeye!

**Pull request merged**